docs/sdk/access-control/encryption.md

```diff
@@ -97,124 +197,374 @@ const accessControlConditions = [
+await contractClient.connect();
+```
+
+After you’ve set your wallet, your next step is to mint the NFT:
 
-```js
-const authSig = await LitJsSdk.checkAndSignAuthMessage({ chain: "ethereum" });
+```jsx
+// this identifier will be used in delegation requests. 
+const { capacityTokenIdStr } = await contractClient.mintCapacityCreditsNFT({
+  requestsPerKilosecond: 80,
+  // requestsPerDay: 14400,
+  // requestsPerSecond: 10,
+  daysUntilUTCMidnightExpiration: 2,
+});
 ```
 
-#### Obtaining the Decrypted Data
+In the above example, we are configuring 2 properties:
+
+- `requestsPerDay` - How many requests can be sent in a 24 hour period.
+- `daysUntilUTCMidnightExpiration` - The number of days until the nft will expire. expiration will occur at `UTC Midnight` of the day specified.
 
-In the example, we used `encryptString()` to encrypt so we will use `decryptToString()` to decrypt. Pass in the data `accessControlConditions`, `ciphertext`, `dataToEncryptHash`, `authSig`.
+Once you mint your NFT you will be able to send X many requests per day where X is the number specified in `requestsPerDay`. Once the `Capacity Credit` is minted the `tokenId` can be used in delegation requests.
+
+#### Delegate usage of your NFT
+
+Once you have minted a Capacity Credits NFT, you can delegate usage of it to the PKP we minted earlier. This will allow the delegatee address(es) to use it to make requests to the Lit nodes (in this case, to make a decryption request).
+
+```jsx
+const { capacityDelegationAuthSig } =
+  await litNodeClient.createCapacityDelegationAuthSig({
+    uses: '1',
+    signer: wallet,
+    capacityTokenId: capacityTokenIdStr,
+    delegateeAddresses: [walletAddress],
+  });
+```
+
+To delegate your Rate Limit NFT there are 4 properties to configure:
+
+- `uses` - How many times the delegation may be used
+- `dAppOwnerWallet` - The owner of the wallet as an `ethers Wallet instance`
+- `capacityTokenId` - The `token identifier` of the Rate Limit NFT
+- `delegateeAddresses` - The wallet addresses which will be delegated to
 
 :::note
-If you want to use another LitJsSDK encryption method to encrypt content, you will need to use the appropriate decrypt method.
+The `delegateeAddress` parameter is optional. If omitted, anyone can use your `capacityDelegationAuthSig` to use your app without restrictions. In this case, you can utilize other restrictions like the `uses` param to limit the amount of usage by your users.
 :::
 
-```js
-const decryptedString = await LitJsSdk.decryptToString(
-  {
-    accessControlConditions,
-    ciphertext,
-    dataToEncryptHash,
-    authSig,
-    chain: "ethereum",
-  },
-  litNodeClient
-);
+Check out a complete example [here](https://github.com/LIT-Protocol/js-sdk/blob/1286138adc09ac2d34371f3ac12a9088ada367ec/e2e-nodejs/group-rli/test-rli-from-lit-node-client-diff-delegatee.mjs).
+
```

Comment:
We are removing the `e2e` nodejs tests soon. so perhaps we just remove this link and update once we merge in `auth unification` with the new `local-test` test cases?